main/swadgeMode.h

```diff
@@ -126,4 +126,7 @@ typedef struct _swadgeMode
     void (*fnEspNowSendCb)(const uint8_t* mac_addr, esp_now_send_status_t status);
 } swadgeMode;
 
-#endif
\ No newline at end of file
+void overrideToSwadgeMode( swadgeMode* mode );
+void switchToSwadgeMode(uint8_t mode);
```

Comment:
Fine here, but remove it from `mode_main_menu.h` please. No one needs duplicates!